general_tests/sessions_rpccaching_it_test.go

```diff
@@ -784,6 +790,98 @@ func testRPCMethodsCdrsStoreSessionCost(t *testing.T) {
 	}
 }
 
+// Load the tariff plan, creating accounts and their balances
```

Comment:
Please rename the file to rpccaching_it_test.go if you are testing more than sessions inside.